.github/workflows/docker-build-and-publish.yml

```diff
@@ -0,0 +1,110 @@
+###############################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+---
+
+name: Build - Docker image Backend (SemVer)
+
+on:
+  push:
+    branches:
+      - main
+    # trigger events for SemVer like tags
+    tags:
+      # match e.g. 1.1.1 and 1.1.1-rc1
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-*'
+  pull_request:
+    branches:
+      - main
+
+env:
+  IMAGE_NAMESPACE: "tractusx"
+  IMAGE_NAME: "ssi-dim-wallet-stub"
+
+jobs:
+  docker-backend:
+    if: github.repository_owner == 'eclipse-tractusx' # prevent running on forks
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      -   name: Checkout
+          uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      # Copy legal information from root dir to backend dir for publishing
+
+      -   name: Copy and paste legal information
```

Comment:
https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/actions/runs/10898362292/job/30241435778?pr=11
are you sure that step makes sense?

Reply:
Removed as this is unnecessary